app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayStore.kt

```diff
@@ -19,7 +19,7 @@ import org.mozilla.fenix.tabstray.browser.TabGroup
  * @property mode Whether the browser tab list is in multi-select mode or not with the set of
  * currently selected tabs.
  * @property inactiveTabs The list of tabs are considered inactive.
- * @property searchTermGroups The list of tab groups.
+ * @property tabPartitions The map of tab partitions.
```

Comment:
The docs aren't right, you're not passing a map here, but a single partition? Shouldn't it be a map?

Reply:
Correct, I'll correct the doc.  Thanks